test/runtests.jl

```diff
@@ -54,7 +54,7 @@ Empty()
 @qstruct Slurp(x, y=1, args...; kwargs...)
 s = Slurp(1,2,3,4,5,6,7; x=1, y=10+2)
 @test s.args == (3,4,5,6,7)
-@test s.kwargs == [(:x, 1), (:y, 12)]
+@test s.kwargs == [(:x => 1), (:y => 12)]
```

Comment:
I think this is the only change w.r.t the API.  It's also possible to stick with previous tuples-of-array API.  But Julia >= 0.7 has pairs in `kwargs` so I thought you might want to change it at some point anyway.  What do you think?

Reply:
It's definitely nicer as pairs.